Fix `NavigateEvent` & `NavigationCurrentEntryChangeEvent` constructor… (#37934)
mdn/content: files/en-us/web/api/navigationcurrententrychangeevent/navigationcurrententrychangeevent/index.md

### Parameters

- `type`
  - : A string representing the type of event.
- `init`

>   - : An object that, _in addition of the properties defined in {{domxref("Event/Event", "Event()")}}_, can have the following properties:

Reviewer: ```suggestion
  - : An object that, _in addition to the properties defined in {{domxref("Event/Event", "Event()")}}_, has the following properties:
```
